ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -247,19 +242,28 @@ open class TableViewCell: UITableViewCell {
     /// A constant representing the number of lines for a label in which no change will be made when the `preferredContentSizeCategory` returns a size greater than `.large`.
     @objc public static let defaultNumberOfLinesForLargerDynamicType: Int = -1
 
+    /// The default horizontal spacing in the cell.
+    @objc public static let horizontalSpacing: CGFloat = 16
+
```

Comment:
Like I mentioned on the other review: do we need to expose those default values as part of the API?

Reply:
The idea is to allow the customer to reset the properties to their default values.

Reply:
Also, in this case, the default values do need to be exposed since they are default params for the static functions above.